web/templates/pages/setup_webapp.html

```diff
@@ -33,7 +33,7 @@
 				<?php if( !$WebappInstaller->isDomainRootClean()): ?>
 					<span class="alert alert-info alert-with-icon">
 						<i class="fas fa-info"></i>
-						<?=_('Data loss warning!<br>Your web domain already has files uploaded, the installer will overwrite your files and/or the installation might fail.<br/><br/> Please use the installer only for empty web domains');?>
+						<?=_('Data loss warning!<br>HESTIA found some files in your domain folder. The installer will overwrite your files and / or the installation might fail.<br/><br/> Please use the installer only for empty domain folders!');?>
```

Comment:
Ideally you want to show:

Data loss warning!
/home/user/web/domain/public_html/ is not empty. The installer will overwrite your files and / or the installation might fail.<br/><br/> Please use the installer only for empty domain folders!');?>

Reply:
I thought that clients do not have access to home but only to their directory, it makes the most sense.

What do you think?

Reply:
It depends if they login with Additional ftp or sftp or file manager 

In both last cases the are able to /web/domain/public_html/ 

In the ftp case it depends on how it is set up by default the user get access to /public_html/